### PR TITLE
PowerShell Password Policy Query

### PR DESCRIPTION
**Pull request by crowface28**
**Details:**
Added Powershell password policy query to T1201 .md and .yaml.

**Testing:**
PS C:\Users\me> Get-ADDefaultDomainPasswordPolicy

ComplexityEnabled           : True
DistinguishedName           : DC=nonya,DC=com
LockoutDuration             : 00:05:00
LockoutObservationWindow    : 00:02:00
LockoutThreshold            : 3
MaxPasswordAge              : 180.00:00:00
MinPasswordAge              : 00:00:00
MinPasswordLength           : 15
objectClass                 : {domainDNS}
objectGuid                  : 12345678-1234-1234-1234-123456789abc
PasswordHistoryCount        : 2
ReversibleEncryptionEnabled : False

**Associated Issues:**

### CODE REVIEW

**Comment by caseysmithrc** [comment]
I think there are a couple things missing here.
One, if this requires a dependency on Windows RSAT, perhaps you can add a link in the description to where to find this.
Two, the YAML file isn't conforming to the expected template. 

Let us know if you have questions or want help to correct.  But you are missing several properties in the test we are expecting.

File: atomics/T1201/T1201.yaml [hunk 1]
```diff
@@ -52,3 +52,15 @@ atomic_tests:
     name: bash
     command: |
       cat /etc/login.defs
+      
+ - name: Examine password policy - Windows
+  description: |
```
Comment (zacbrown) [inline atomics/T1201/T1201.yaml]: Your indentation isn't quite right here. It's causing the doc generation to fail.

Looks like you're off by one space.

Comment (zacbrown) [reply atomics/T1201/T1201.yaml]: Note, it looks like it's an indentation issue all the way down the changes.

File: atomics/T1201/T1201.md [hunk 2]
```diff
@@ -90,3 +90,16 @@ Lists the password expiration policy to console on CentOS/RHEL/Ubuntu.
 cat /etc/login.defs
 ```
 <br/>
+<br/>
+
```
Comment (zacbrown) [inline atomics/T1201/T1201.md]: You don't actually need to update this file. You can remove it from the change.

Comment (crowface28) [reply atomics/T1201/T1201.md]: Not sure what this means, I thought this is where the description and such went?

I'll leave it in after adding requested link to RSAT tools.  Let me know if I *must* remove to have change approved.

File: atomics/T1201/T1201.yaml (hunk 1 above)
Comment (crowface28) [reply atomics/T1201/T1201.yaml]: Tried using online editor, huge fail.  Should be fixed now.

**Comment by clr2of8** [comment]
closing until updated.

**Pull request closed without merging**